documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -79,7 +80,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<www.example.com>/sdk` for vSphere or `https://<www.example.com>/ovirt-engine/api/` for {rhv-short}.
+<2> Specify the API end point URL, for example, `https://<fqdn>/sdk` for vSphere or `https://<fqdn>/ovirt-engine/api/` for {rhv-short}.
```

Comment:
@mguetta1 Do you approve?

Reply:
@RichardHoch please remove the "fqdn" change, it is handled in other PR & BZ